lib/vsc/install/shared_setup.py

```diff
@@ -1357,6 +1357,7 @@ def action_target(target, setupfn=setup, extra_sdist=[], urltemplate=None):
             'setuptools'
         ],
         'excluded_pkgs_rpm': [], # vsc-install ships vsc package (the vsc package is removed by default)
+        'namespace_packages': ['vsc'],
```

Comment:
well, I didn't understand your remark w.r.t. 'pure' namespace; if you spread `vsc` across multiple locations, you need this, and this applies to `vsc-install` too, since it provides stuff accessible through `from vsc.install import ...`

How would I set this as the default for all other vsc packages?

Anyway, I don't think that matters much, we need this for `vsc-install` itself too imho.